src/Jira/Api.php

```diff
@@ -198,6 +198,17 @@ public function editIssue($issue_key, array $params)
 	}
 
 	/**
+	 * Gets attachments meta information.
+	 *
+	 * @return array
+	 * @since  2.0.0
+	 */
+	public function getAttachmentsMetaInformation()
+	{
+		return $this->api(self::REQUEST_GET, '/rest/api/2/attachment/meta', array());
```

Comment:
Isn't the 3rd argument default value also `array()`? If that is so, then you can just omit it.